docs/en/Best-Practices/PostgreSQL-Integration.md

```diff
@@ -0,0 +1,61 @@
+﻿## Entity Framework Core PostgreSQL Integration
+
+> See [Entity Framework Core Integration document](../Entity-Framework-Core.md) for the basics of the EF Core integration.
+
+### EntityFrameworkCore Project Update
+
+- In `Acme.BookStore.EntityFrameworkCore` project replace package `Volo.Abp.EntityFrameworkCore.SqlServer` with `Volo.Abp.EntityFrameworkCore.PostgreSql` 
+- Update to use PostgreSQL in `BookStoreEntityFrameworkCoreModule`. Example:
+
+````C#
+[DependsOn(
+        //code omitted for brevity
+		
+		/* This was updated from AbpEntityFrameworkCoreSqlServerModule */
+        typeof(AbpEntityFrameworkCorePostgreSqlModule),
+		/* This was updated from AbpEntityFrameworkCoreSqlServerModule */
+        
+		//code omitted for brevity
+        )]
+public class Acme.BookStore.EntityFrameworkCoreModule : AbpModule
+    {
+        public override void ConfigureServices(ServiceConfigurationContext context)
+        {
+            context.Services.AddAbpDbContext<BookStoreDbContext>(options =>
+            {
+                options.AddDefaultRepositories(includeAllEntities: true);
+            });
+
+            Configure<AbpDbContextOptions>(options =>
+            {
+                /* This was updated */
+                options.UsePostgreSql();
+				/* This was updated */
+            });
+        }
+    }
+````
+
+### EntityFrameworkCore.DbMigrations Project Update
```

Comment:
Removing the existing migration and then re-adding the migration seems to be simpler and less error-prone.